internal/objects/lua_object_zone.go

```diff
@@ -0,0 +1,121 @@
+package objects
```

Comment:
This file is now auto generated, you should move your `LoadMOBFromConfig` function into the `Zone` struct (`object_zone.go`) and then re-run `go generate ./...`.

Delete this file when you have moved your method over.

This will require a small change to the function but it'll be something like this:

```
// Added by Tim for loading in mobs
func (z *Zone) LoadMOBFromConfig(mobIDstring) *Unit {
	mobConfig, ok := z.BaseConfig.MOBs[strings.ToLower(mobID)]

	if !ok {
                // log error here, there is no support for returning errors right now
		// "could not find mob config with ID: " + mobID
                return
	}

	mob := NewMOBFromConfig(mobConfig)
        
        return mob
}
```